src/main/java/com/google/flourbot/entity/action/ActionType.java

```diff
@@ -0,0 +1,3 @@
+package com.google.flourbot.entity.action;
+
+public enum ActionType { SHEET_APPEND }
```

Comment:
Enum of all possible action types.

Reply:
Add a TODO to make it clear that you plan to expand it.

Reply:
Done!